site/sr_SR.all.json

```diff
@@ -1082,10 +1082,12 @@
     "other": "Цена"
   },
   "shopping_price_plan_note": {
-    "zero": "Наплаћује се сваки {{ .Interval }} .",
     "one": "Наплаћује се сваки {{ .Interval }} након завршетка бесплатног пробног периода од 24 сата.",
```

Comment:
Serbian requires [one, few, and other](https://github.com/nicksnyder/go-i18n/blob/0dc1626d56435e9d605a29875701721c54bc9bbd/i18n/language/codegen/plurals.xml#L108).